roles/node/tasks/get_genesis_configs.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Clone the repository to /tmp
+  ansible.builtin.git:
+    repo: "{{ vars[project][type].configs_source_url }}"
+    dest: "{{ tmp_dir }}/{{ project }}"
+    version: "{{ vars[project][type].configs_version }}"
+    force: true
+
+- name: Delete rollup toml file
+  ansible.builtin.file:
+    path: "{{ tmp_dir }}/{{ project }}/configs/celestia/rollup.toml"
+    state: absent
```

Comment:
lets add a comment that we are deleting it because we are maintaining a copy of it in toml.js